engine/orders.go

```diff
@@ -351,6 +351,18 @@ func (o *orderManager) Submit(newOrder *order.Submit) (*orderSubmitResponse, err
 	if exch == nil {
 		return nil, ErrExchangeNotFound
 	}
+
+	// Checks for exchange min max limits for order amounts before order
+	// execution can occur
+	err := exch.CheckOrderExecutionLimits(newOrder.AssetType,
+		newOrder.Pair,
+		newOrder.Price,
+		newOrder.Amount,
+		newOrder.Type)
+	if err != nil {
+		return nil, err
```

Comment:
It would be good to decorate this error with the exchange name since this function does not.